source/setup/nao_setup.rst

```diff
@@ -105,3 +105,33 @@ Finally, for all robots, run
     PC$ nao_sync -rd treebeard
     # reboot again
     
+*********************************
+Connecting to GameController Wifi
+*********************************
+
+To play a game by the SPL Rules, the `runswift` executable needs to send packets of 
+information to, and respond to commands from, one specific soccer field's 
+`GameController <https://github.com/RoboCup-SPL/GameController3/>`_. Typically the
+competition organisers will provide the list of field SSIDs and any other details 
+at the competition, for example `SPL_A` to `SPL_E` has been typical of RoboCup.
+
+One way to set this up is to SSH into the robot and use the change field script, this is 
+analagous to connecting to a wifi hotspot, one needs to specify things like the 
+`SSID <https://www.lifewire.com/definition-of-service-set-identifier-816547>`_:
+
+```
+    PC$ ssh nao@treebeard.local
+    # sudo bin/changeField.py <SSID>
+    sudo bin/changeField.py SPL_E
+```
+
+These could also have been set from the `runswift.cfg` file.
```

Comment:
```suggestion
```
Having wifi related stuff in runswift.cfg when the runswift binary doesn't use it at all doesn't feel right. runswift.cfg should be for binary stuff only